### ISSUE TITLE
RouteAlreadyExists error 

### ISSUE THREAD

**Issue by szinck1** [terraform-aws-modules/terraform-aws-vpc#108]
I'm running into the same(?) error as issue #44.

$ terraform version
Terraform v0.11.5
+ provider.aws v1.11.0

I successfully ran the same code in a different Region and AZ's previous to this execution... 

My code:
```
module "vpc" {
  source = "terraform-aws-modules/vpc/aws"

  name = "Staging VPC US-EAST"
  cidr = "10.0.0.0/16"


 azs             = ["us-east-1a", "us-east-1b", "us-east-1c" ]
  private_subnets = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
  public_subnets  = ["10.0.101.0/24", "10.0.102.0/24", "10.0.103.0/24"]

  enable_nat_gateway = true
  enable_vpn_gateway = true

  tags = {
    Terraform = "True"
    Environment = "Staging"
  }
}

```

If I run it, everything looks good until:
```
* module.vpc.aws_route.private_nat_gateway[2]: 1 error(s) occurred:

* aws_route.private_nat_gateway.2: Error creating route: RouteAlreadyExists: The route identified by 0.0.0.0/0 already exists.
	status code: 400, request id: 879d33bf-6804-4e74-94a2-7a1e7c3c3040
* module.vpc.aws_route.private_nat_gateway[1]: 1 error(s) occurred:

* aws_route.private_nat_gateway.1: Error creating route: RouteAlreadyExists: The route identified by 0.0.0.0/0 already exists.
	status code: 400, request id: aa381109-6e9d-4902-9c3c-36f25a80de9e
* module.vpc.aws_route.private_nat_gateway[0]: 1 error(s) occurred:

* aws_route.private_nat_gateway.0: Error creating route: RouteAlreadyExists: The route identified by 0.0.0.0/0 already exists.
	status code: 400, request id: 50f82579-0fbb-4ff6-9831-352f00a3bb54
```

**Comment by szinck1** [comment]
It worked after I deleted the default VPC. But why would that cause an issue? They're on different IP networks.  Also, the same code worked in `ca-central-1` and the default VPC still exists.

**Comment by horsey** [comment]
@szinck1  The issue in #44 seems to be a corner case that was fixed by @antonbabenko. However, I am still seeing this issue intermittently.  I do not think it's related to the default VPC because I see the error even after the default VPC is deleted.

**Comment by antonbabenko** [comment]
Thanks, everyone for reporting this issue. I am quite certain that this issue is now fixed in #111 and [v1.27.0](https://github.com/terraform-aws-modules/terraform-aws-vpc/releases/v1.27.0) has been released.

Please give it a try and leave a comment if the problem still exists.
